source/hic_hal/stm32/stm32h743ii/extended_features/source/aux_task.c

```diff
@@ -0,0 +1,64 @@
+#include "aux_task.h"
+#include "util.h"
+#include "adapter_adc.h"
+
+#ifndef USE_LEGACY_CMSIS_RTOS
```

Comment:
where is this flag from? if noone would build UDB with this legacy cmsis os, then we don't need these flags and the file is more readable.